docs/data-sources/pg.md

```diff
@@ -95,6 +95,7 @@ Read-Only:
 - **admin_username** (String)
 - **backup_hour** (String)
 - **backup_minute** (String)
+- **enable_ipv6** (String)
```

Comment:
Hmm, shouldn't it be `Boolean`?

Reply:
This is a known issue: #620 , relates to https://github.com/aiven/terraform-provider-aiven/blob/3a8c53ea153010145c4967d3a889ad8fc94cb899/aiven/internal/uconf/user_config.go#L53 

Id think we should ignore it for now and fix the user_config generation at some point.